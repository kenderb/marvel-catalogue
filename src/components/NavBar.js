import React from 'react';
import { Link } from 'react-router-dom';

const FilterComics = () => (
  <header>
    <nav>
      <ul>
        <li>
          <Link to="/"> LOGO </Link>
        </li>
        <li>
          <Link to="/"> List </Link>
        </li>
      </ul>
    </nav>
  </header>
);

export default FilterComics;
