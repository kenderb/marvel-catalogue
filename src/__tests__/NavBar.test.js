import { BrowserRouter } from 'react-router-dom';
import { create } from 'react-test-renderer';
import NavBar from '../components/NavBar';

test('NavBar renders correctly', () => {
  const tree = create(
    <BrowserRouter>
      <NavBar />
    </BrowserRouter>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
