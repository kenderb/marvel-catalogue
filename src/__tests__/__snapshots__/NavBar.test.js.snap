// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar renders correctly 1`] = `
<header>
  <nav>
    <ul>
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
           LOGO 
        </a>
      </li>
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
           List 
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
